base/applications/taskmgr/applpage.c

```diff
@@ -512,14 +512,14 @@ void ApplicationPageUpdate(void)
     {
         EnableWindow(hApplicationPageEndTaskButton, FALSE);
     }
-    /* Enable "Switch To" button only if one app is selected */
+    /* Enable "Switch To" button only if only one app is selected */
     if (ListView_GetSelectedCount(hApplicationPageListCtrl) == 1 )
     {
         EnableWindow(hApplicationPageSwitchToButton, TRUE);
     }
     else
     {
-    EnableWindow(hApplicationPageSwitchToButton, FALSE);
+        EnableWindow(hApplicationPageSwitchToButton, FALSE);
     }
```

Comment:
Why not replace this entire block with just:
`EnableWindow(hApplicationPageSwitchToButton, ListView_GetSelectedCount(hApplicationPageListCtrl) == 1);`

Reply:
Yes good idea !